app/views/users/show.html.erb

```diff
@@ -1,11 +1,29 @@
-<h1>This page shows a Selected user's profile</h1>
-<ul>It shows :
-<li>User's Name</li>
-<li>User's Biography</li>
-<li>Number of all user's posts</li>
-<li>3 of User's posts</li>
-<li>Number of likes on each post</li>
-<li>Number of comments on each post</li>
-<li>button to see all post</li>
+<ul>
+    <li class="user-card">
+        <div class="img-holder"><%= image_tag @user.photo %></div>
+        <div class="card-details">
+        <h4 class="user-name"><%= @user.name %></h4>
+        <p class="num-of-posts">Number of posts: <%= @user.post_count %></p>
+        </div>
+    </li>
+    <br/>
+    <li class="user-card">
+        Bio
+        <p class="bio"><%= @user.bio %></p>
+        </div>
+    </li>
+    <% @user_recent_posts.each do |post| %>
+        <li class="user-card">
+            <div class="card-details">
+            <h6 class="post-title">Post <%= post.id %></h6>
+            <p class="post-brief"><%= post.text %></p>
+            <p class="num-of-posts">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+            </div>
+        </li>
+    <br/>
+    <% end %>
+</ul>
+<%= link_to user_posts_path(@user), class: 'button' do %>
+  Show All
```

Comment:
- [ Required ] I think it would be nice if you center the elements on this page so that the margins on the `left` and `right` sides are equal. Currently, the margins on your design are not equal one is bigger than the other. If you look at the project wireframe for this milestone the margins are equal. Please see the screenshots below and rectify this.

## Your design
![Screenshot from 2023-09-15 02-28-56](https://github.com/danielochuba/duo-blog/assets/16392046/1f337072-18cc-4a69-871e-b9c70bde3e82)


## Project wireframe
![](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAABTYBUDYSDDM5F55VB2WGP54ZIDUSVQ)